packages/config/config/devices/0x013c/pat02-a.json

```diff
@@ -0,0 +1,338 @@
+	"paramInformation": {
+		"1": {
+			"label": "Basic Set Dim Level (Off)",
+			"description": "When the flood trigger is off, send this dim level via the Basic CC to Group 2.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Basic Set Dim Level (On)",
+			"description": "When the flood trigger is on, send this dim level via the Basic CC to Group 2.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"5[0x00]": {
+			"label": "Flood Sensor Function",
+			"description": "Enable or disable the flood function.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 1
+				},
+				{
+					"label": "Enabled",
+					"value": 0
+				}
+			]
+		},
+		"5[0x03]": {
+			"label": "Temperature Scale",
+			"description": "Configure the temperature scale reported by the device.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Fahrenheit",
+					"value": 0
+				},
+				{
+					"label": "Celsius",
+					"value": 1
+				}
+			]
+		},
+		"5[0x05]": {
+			"label": "Temperature Report When Triggered",
+			"description": "Determines whether a temperature report is sent after an event is triggered.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 1
+				},
+				{
+					"label": "Enabled",
+					"value": 0
+				}
+			]
+		},
+		"6[0x06]": {
+			"label": "Humidity Report When Triggered",
+			"description": "Determines whether a temperature report is sent after an event is triggered.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 1
+				},
+				{
+					"label": "Enabled",
+					"value": 0
+				}
+			]
+		},
+		"7[0x03]": {
+			"label": "Disable Basic Off after flood",
+			"description": "Determines whether a Basic Off event is sent after a flood event is cleared.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 1
+				},
+				{
+					"label": "Enabled",
+					"value": 0
+				}
+			]
+		},
+		"7[0x04]": {
```

Comment:
Bit 4? That needs to be 0x10 then

Reply:
Value 16 set binary report

Reply:
Ditto. Where are you getting 16? The manual says Bit 4.